feed/config.py

```diff
@@ -18,31 +19,21 @@ class Config:
 
```

Comment:
Maybe increment VERSION = "0.3" ?

Reply:
up to 0.4 or would this count as 1.0 since its theoretically getting released?